### PR TITLE
Add draft of dask-on-summit post

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: _posts/2019-08-26-dask-on-summit.md [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+
+mrocklin@my-laptop $ ssh -L 8888:login2:8888 summit.olcf.ornl.gov
+```
+
+I can now connect to Jupyter from my laptop by navigating to
+http://localhost:8888 , run the cluster commands above in a notebook, and
+things work great.  Additionally, thanks to `jupyter-server-proxy`, Dask's
+dashboard is also available at http://localhost:8888/proxy/####/status , where
+`####` is the port currently hosting Dask's dashboard.  You can probably find
+this by looking at `cluster.dashbaord_link`.  It defaults to 8787, but if
```
Comment (rmccorm4) [inline _posts/2019-08-26-dask-on-summit.md]: > `cluster.dashbaord_link`

`cluster.dashboard_link`

File: _posts/2019-08-26-dask-on-summit.md [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+#BSUB -n 128
+#BSUB -R "span[hosts=1]"
+#BSUB -M 600000
+#BSUB -W 00:30
+JOB_ID=${LSB_JOBID%.*}
+
+/ccs/home/mrocklin/anaconda/bin/python -m distributed.cli.dask_worker tcp://scheduler:8786 --nthreads 16 --nprocs 8 --memory-limit 75.00GB --name name --nanny --death-timeout 60 --interface ib0 --interface ib0
+```
+
+After comparing notes with existing systems, we modify add keywords to add and
```
Comment (rmccorm4) [inline _posts/2019-08-26-dask-on-summit.md]: > we modify add keywords

typo?

File: _posts/2019-08-26-dask-on-summit.md [hunk 3]
```diff
@@ -0,0 +1,446 @@
...
+Dask.  Dask is conveniently deployable from other Cloud, Hadoop, and local
+systems.  See the [Dask setup
+documentation](https://docs.dask.org/en/latest/setup.html) for more
+information.
+
+
+### Future work: GPUs
+
+Summit is fast because it has a ton of GPUs. I'm going to work on that next,
+but that will probably enough content to fill up a whole other blogpost :)
```
Comment (rmccorm4) [inline _posts/2019-08-26-dask-on-summit.md]: > will probably enough content

will probably {be, cover}? enough content

File: _posts/2019-08-26-dask-on-summit.md [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+        # by taking the global lock
+        try:
+                with workspace._global_lock():
+                    self._lock_file = locket.lock_file(self._lock_path)
+                    self._lock_file.acquire()
+```
+
+It looks like Dask is trying to use a file-based lock.
+Unfortunately some NFS systems don't like file-based locks, or handle them very
+slowly.  Looking up the `is_locking_enabled` function we see that it checks a
```
Comment (maximumcats) [inline _posts/2019-08-26-dask-on-summit.md]: In the case of Summit, OLCF policy disables write access to the login nodes (where the home directory is NFS-mounted) from the compute nodes. So I recommend amending this to say:

"Unfortunately some NFS systems don't like file-based locks, or handle them very slowly. In the case of Summit, the home directory is actually mounted read-only from the compute nodes, so a file-based lock will simply fail."

**Pull request merged**